liquibase-core/src/main/java/liquibase/change/core/CreateProcedureChange.java

```diff
@@ -22,16 +22,7 @@
 import java.io.UnsupportedEncodingException;
 import java.util.Map;
 
-@DatabaseChange(
-    name = "createProcedure",
```

Comment:
Just verifying ... we do **not** want to keep this in the documentation? I think removing it makes sense but I want to double-check it was intentional.

Reply:
Intentional! Too long for this file. We have pretty much this exact text on the docs.